### PR TITLE
update proxy cache sizes and set upper limit

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR should prevent proxy cache from running out of space. Assuming cache volume is 100GB calculation done in https://trello.com/c/r9yTDj0y/82-outage#comment-582ee489a678bf8e6aa742dc

cc: @joshmoore @manics @eleanorwilliams

### CODE REVIEW

**Comment by joshmoore** [comment]
Nice. No objections from my side.

**Comment by aleksandra-tarkowska** [comment]
could someone double check numbers in the spreadsheet, please?

**Comment by manics** [comment]
This merges fine onto my branch. Can you update the README with the new parameter: https://github.com/openmicroscopy/infrastructure/blame/master/ansible/roles/nginx-proxy/README.md#L86

**Comment by aleksandra-tarkowska** [comment]
done

File: ansible/roles/nginx-proxy/README.md [hunk 1]
```diff
@@ -103,7 +104,7 @@ Caching:
 - `nginx_proxy_cache_lock_time`: Prevent multiple backend requests to the same object (subsequent requests will wait for the first to either return or time-out), default 1 minute
 - `nginx_proxy_cachebuster_port`: An alternative port which can be used to force a cache refresh, disabled by default. You should ensure this is firewalled. If SELinux is enabled and the port is not one that nginx can bind by default (typically 80, 81, 443, 488, 8008, 8009, 8443, 9000 are allowed by default) you must update your policy yourself.
 
-Warning: `max_size` is not set on any disk caches, so you should put `nginx_proxy_cache_parent_path` on a separate partition.
+Warning: for conviniounce, put `nginx_proxy_cache_parent_path` on a separate partition (calculate size of the partition based on `max_size` set on disk caches).
```
Comment (manics) [inline ansible/roles/nginx-proxy/README.md]: typo: `conviniounce` -> `convenience`

**Comment by aleksandra-tarkowska** [comment]
done

**Pull request merged**